src/pages/bioclimatic/index.js

```diff
@@ -24,10 +25,13 @@ const Container = () => {
   const { country } = (match && match.params) || {};
   const currentQueryParams = useQueryParams();
   const { startYear, endYear, scenario } = currentQueryParams;
+  const [viewport, setViewport] = useState({ zoom: 4, latitude: 40, longitude: -5 });
```

Comment:
Does it need a default viewport? Just curiosity

Reply:
I think you're right, but I think it's still better than `null`